rpcserver.go

```diff
@@ -1663,7 +1677,7 @@ func (r *rpcServer) SendPaymentSync(ctx context.Context,
 		Target:      destPub,
 		Amount:      amtMSat,
 		PaymentHash: rHash,
-	})
+	}, nil)
```

Comment:
Realized I should have a non-nil maxFee here too...
This means I will need to do the type switch again, so I think I should make it a function. Where would be a good place to put a utility function like that?